govc/vm/guest/guest.go

```diff
@@ -70,44 +68,18 @@ func (flag *GuestFlag) Process(ctx context.Context) error {
 	return nil
 }
 
-func (flag *GuestFlag) Toolbox() (*toolbox.Client, error) {
-	pm, err := flag.ProcessManager()
-	if err != nil {
-		return nil, err
-	}
-
-	fm, err := flag.FileManager()
-	if err != nil {
-		return nil, err
-	}
-
+func (flag *GuestFlag) Toolbox(ctx context.Context) (*toolbox.Client, error) {
```

Comment:
This is a breaking change, right? How should we handle this? Not sure how many users are using this API, we could flag it in the release notes as such.

Reply:
Users aren't supposed to use any of the govc command packages directly and are own their own if they do :)
Outside of govc/ , I agree we should doc any breaking changes.